packages/flutter/lib/src/material/drawer.dart

```diff
@@ -264,10 +281,28 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   final ColorTween _color = new ColorTween(begin: Colors.transparent, end: Colors.black54);
   final GlobalKey _gestureDetectorKey = new GlobalKey();
 
+  AlignmentDirectional get drawerStartAlignment {
+    switch(widget.type) {
+      case DrawerAlignment.START: return AlignmentDirectional.centerStart;
```

Comment:
The return should go on the next line.  Also, no need for `break` after `return`.

Reply:
Fixed